kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/ProjectAuthoringIntegrationTest.java

```diff
@@ -33,19 +35,24 @@
 
     private static final Logger LOG = LoggerFactory.getLogger( ProjectAuthoringIntegrationTest.class );
 
+    private HomePerspective home;
+
+    @Before
+    public void setup() {
```

Comment:
@jhrcek Hi, since it's a test for Authoring and the only way to access authoring is if some accessible project already exists in the workbench, I think it's fair that the `setup` method guarantees that there is a project and the tests can access Authoring. :)

Also, @bsremac will work on this tests, maybe we can have some improvements in that aspect. :)